### PR TITLE
[Access-tokens][ClusterLoader2] Add standalone test

### PR DESCRIPTION
**Pull request by jprzychodzen**
Add standalone test for ClusterLoader2 that allows benchmarking and
testing access tokens thresholds.

This test is based on Mateusz's code from github.com/mm4tt/k8s-util.

### CODE REVIEW

File: clusterloader2/testing/access-tokens/config.yaml [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Stress testing access token validation
+#
+# Targeting 2 000 tokens with 20 000 total QPS
```
Comment (mm4tt) [inline clusterloader2/testing/access-tokens/config.yaml]: Ideally, the test should scale with the cluster size (as load test does). In such case we'll be able to run it along our CI/CD tests at 100, 500 and 5k node scale.

Comment (jprzychodzen) [reply clusterloader2/testing/access-tokens/config.yaml]: Added linear scaling of QPS. Actually, we could get more QPS for smaller clusters, but this sets reasonable requirements and expectations what can be delivered.

**Comment by jprzychodzen** [comment]
Decreased QPS target number to match real-world expectations

File: clusterloader2/testing/access-tokens/config.yaml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+name: access-tokens
+automanagedNamespaces: {{$namespaces}}
+tuningSets:
+  - name: Sequence
+    parallelismLimitedLoad:
+      parallelismLimit: 1
+steps:
+- name: Starting measurements
+  measurements:
+    - Identifier: APIResponsivenessPrometheus
```
Comment (mm4tt) [inline clusterloader2/testing/access-tokens/config.yaml]: Please also start the TestMetrics measurement (you can see how it's done in density or load test).
It's a bundle measurement producing a lot of useful data (e.g. profiles, restart checks, resource usages, etc.)

File: clusterloader2/testing/access-tokens/config.yaml [hunk 3]
```diff
@@ -0,0 +1,137 @@
+# Stress testing access token validation
+#
+# Targeting 2 000 tokens with 5 000 total QPS for 5k node cluster, so it's 2.5 QPS per token.
+# For smaller cluster, we want to scale down lineary QPS per token to 2.5 * (Number of nodes)/(5 000).
```
Comment (mm4tt) [inline clusterloader2/testing/access-tokens/config.yaml]: Could you make it more explicit that this test always creates 2K tokens and 80 pods and only the QPS depends on the cluster size?

Comment (jprzychodzen) [reply clusterloader2/testing/access-tokens/config.yaml]: Added comment and calculations.

File: clusterloader2/testing/access-tokens/config.yaml [hunk 4]
```diff
@@ -0,0 +1,167 @@
+# Stress testing access token validation
+#
+# Targeting 2 000 tokens with 5 000 total QPS for 5k node cluster, so it's 2.5 QPS per token.
+#
+# Number of tokens does not scale with number of nodes.
+# By default, those 2 000 tokens are are assigned to 80 service accounts, with 25 tokens each.
```
Comment (mm4tt) [inline clusterloader2/testing/access-tokens/config.yaml]: Sorry for nitpicking, but I'd also add a line about number of pods.
IIUC there will be always 80 pods, which means that this test can be run even on a 1 node cluster (assuming the node is big enough). If we had more pods then we'd run into 110pods/node limitation.

**Pull request merged**